SPECS/rust/rust.spec

```diff
@@ -41,10 +41,13 @@ Source4:        https://static.rust-lang.org/dist/%{release_date}/rust-std-%{sta
 Source5:        https://static.rust-lang.org/dist/%{release_date}/cargo-%{stage0_version}-aarch64-unknown-linux-gnu.tar.xz
 Source6:        https://static.rust-lang.org/dist/%{release_date}/rustc-%{stage0_version}-aarch64-unknown-linux-gnu.tar.xz
 Source7:        https://static.rust-lang.org/dist/%{release_date}/rust-std-%{stage0_version}-aarch64-unknown-linux-gnu.tar.xz
-Patch0:         CVE-2023-45853.patch
-Patch1:         CVE-2024-32884.patch
-Patch2:         CVE-2024-31852.patch
 
+Patch0:		Remove_cannot_write_error_test.patch
+Patch1:		Remove_leave_log_after_failure_test.patch
+Patch2:		Ignore_failing_ci_tests.patch
+Patch3:		skip-failing-run-make-tests.patch
+Patch4:		Ignore-test-for-aarch64.patch
```

Comment:
error: linker `aarch64-linux-gnu-gcc` not found. Let's assess whether this is expected to fail for Azure Linux or e.g. whether the linker binary would need to be renamed.

Side note: Let's add some documentation for all the patch files we add. This way it becomes clear why we did this